Yoast/YoastSEO.js
# Add prominent words for Swedish

## Summary

This PR can be summarized in the following changelog entry:

* Add prominent words for Swedish

## Test instructions

This PR can be tested by following these steps:

*Build a browserfied example
* Set locale to sv_SE.
* Paste a Swedish text into the example.
* Make sure you see no strange word combinations, counts, etc.

Fixes #1797
